BrainPortal/app/controllers/nh_application_controller.rb

```diff
@@ -108,6 +109,9 @@ def prepare_nh_messages
   # This method is identical to (and overrides) the one in
   # ApplicationController excepts it uses the NeuroHub password reset form.
   def check_password_reset #:nodoc:
+    # todo SB - not sure why this needed. to use nh_change_password, user has to type url manually,
```

Comment:
do not leave personal comments in the code

Reply:
remove then